app/src/main/java/com/sun/hero_02/screen/detail/adapter/ItemHeroAbilityViewModel.kt

```diff
@@ -0,0 +1,25 @@
+package com.sun.hero_02.screen.detail.adapter
+
+import androidx.databinding.BaseObservable
+import androidx.databinding.Bindable
+import com.sun.hero_02.data.model.HeroSpell
+import com.sun.hero_02.utils.HeroImageType
+import com.sun.hero_02.utils.HeroSpellSymbol
+
+class ItemHeroAbilityViewModel : BaseObservable() {
+
+    @Bindable
+    var heroSpell: HeroSpell? = null
+        set(value) {
+            field = value
+            notifyChange()
+        }
```

Comment:
Hơn nữa, nếu em gọi hàm notifyChange() như trên, 
=> mỗi lần thay đổi dữ liệu, view sẽ phải bind lại  toàn bộ các trường Bindable, mình chỉ nên notify nguyên trường cần update thôi 

Bây giờ Databinding đã kết hợp với LiveData rồi nên em không phải làm như này nữa 
Em sửa dụng LiveData kết hợp thay cho các trường observer